### ISSUE TITLE
Add additional 433Mhz Codes

### ISSUE THREAD

**Issue by ev0lister** [Homegear/Homegear-Intertechno#5]
Is there any plan to insert more codes the CUL can decode?
Like in FHEM many sensors (temperature, humidity etc.) can be read by the CULs. It would really be great if we can add those codes here as well.

Homegear would really benefit from that as many cheap sensors would still be usable.

Cheers
ev0l

**Comment by hfedcba** [comment]
Of course, it's no big deal - just somebody has to do it ;-). You can already read all CUL packets using Node-BLUE.

**Comment by ev0lister** [comment]
Is there a stub version of a module available? I would like starting implementing just a temperature sensor. I already had a look at the intertechno code, however it seems to be more work editing that rather than starting with a stub.

**Comment by hfedcba** [comment]
Sadly not...

**Comment by ev0lister** [comment]
OK, no worries. From design perspective: Would it make sense to include the other codes/devices into the intertechno module? I'm just checking the code and would most probably create new "Packet" classes for each of them. Dont think it makes sense to create a module for each and every device around there. What do you think?

**Comment by hfedcba** [comment]
If the devices are similar to Intertechno devices, I think we can use the Intertechno module for them.

**Comment by hfedcba** [comment]
> and would most probably create new "Packet" classes for each of them

That would be so great!!! :+1: :smiley:

**Comment by andyboeh** [comment]
I just created a separate Somfy RTS module [here](https://github.com/andyboeh/Homegear-Somfy) that could (better: should) also be integrated into the Intertechno module (I took the Intertechno module as a template anyway). IMHO it would make sense to add additional device decoders at the same time. Do you have a list of devices that you would like to get included and/or sample code? Or did you abandon this issue in the meantime?

**Comment by hfedcba** [comment]
Wow, good job! The issue is still open and our aim is to support as many devices as possible.

I'd say as we can't rename the Intertechno module it would be clearer for other users if the Somfy module stays seperate - even if a lot of the code is the same.

As for missing 433MHz protocols, I think the most important one's already are implemented now - the last really important missing one was Somfy. Or am I missing something? FS20 is not sold anymore. Is there another important family?

**Comment by andyboeh** [comment]
As far as I understood, this issue refers to SlowRF protocols as used by weather stations, wireless door bells, etc. I'm using RFLink on an ESP8266 for this purpose.

It's fine for me if Somfy stays separate - I'd appreciate any input on the Somfy module and I'd be happy if it could be integrated into Homegear when all issues are sorted out.

**Comment by hfedcba** [comment]
> I'd be happy if it could be integrated into Homegear when all issues are sorted out.

That would be really cool!
